src/engraving/libmscore/excerpt.cpp

```diff
@@ -412,13 +427,17 @@ void MasterScore::deleteExcerpt(Excerpt* excerpt)
     undo(new RemoveExcerpt(excerpt));
 }
 
-void MasterScore::initExcerpt(Excerpt* excerpt)
+void MasterScore::initExcerpt(Excerpt* excerpt, bool fakeUndo)
```

Comment:
I don't quite understand the meaning of this flag. Can you please explain it to me?

Reply:
In the original version of `MasterScore::initExcerpt()` it was not possible to undo creating an `Excerpt` but undo/redo creating an `Excerpt` is required in several cases.